src/test/java/org/folio/rest/impl/CodexInstanceResourceImplTest.java

```diff
@@ -41,89 +38,38 @@
-    vertx.deployVerticle(RestVerticle.class.getName(), opt, context.asyncAssertSuccess());
-    RestAssured.port = okapiPort;
-    logger.info("Codex Instances Resource Test Setup Done using port " + okapiPort);
-
+    super.setUp(context);
+    final Async async = context.async();
     final int serverPort = Integer.parseInt(System.getProperty("serverPort", Integer.toString(51234)));
     final String host = "localhost";
-
-    final Async async = context.async();
     final HttpServer server = vertx.createHttpServer();
     server.requestHandler(req -> {
       if (req.path().equals(String.format("/rm/rmaccounts/test/titles/%s", "99999"))) {
         req.response().setStatusCode(200).putHeader("content-type", "application/json")
-            .end(readMockFile(MOCK_RMAPI_INSTANCE_TITLE_200_RESPONSE_WHEN_FOUND));
+          .end(readMockFile(MOCK_RMAPI_INSTANCE_TITLE_200_RESPONSE_WHEN_FOUND));
       } else if (req.path().equals(String.format("/rm/rmaccounts/test/titles/%s", "1"))) {
         req.response().setStatusCode(404).putHeader("content-type", "text/plain")
-            .end(readMockFile(MOCK_RMAPI_INSTANCE_TITLE_404_RESPONSE_WHEN_NOT_FOUND));
+          .end(readMockFile(MOCK_RMAPI_INSTANCE_TITLE_404_RESPONSE_WHEN_NOT_FOUND));
       } else if (req.path().equals("/rm/rmaccounts/test/titles")) {
         if (req.uri().contains("searchfield=titlename&search=Bridget+Jones&orderby=titlename&count=10&offset=1")) {
           req.response().setStatusCode(200).putHeader("content-type", "application/json")
-          .end(readMockFile(MOCK_CODEX_INSTANCE_TITLE_COLLECTION_200_RESPONSE_WHEN_FOUND));
+            .end(readMockFile(MOCK_CODEX_INSTANCE_TITLE_COLLECTION_200_RESPONSE_WHEN_FOUND));
         } else {
           req.response().setStatusCode(500).end("Unexpected call: " + req.path());
         }
       } else {
         req.response().setStatusCode(500).end("Unexpected call: " + req.path());
       }
     });
-
     server.listen(serverPort, host, ar -> {
       async.complete();
     });
-
-    final Map<String, String> okapiHeaders = new HashMap<>();
-    okapiHeaders.put("X-Okapi-Tenant", "codexinstancesresourceimpltest");
-    okapiHeaders.put("X-Okapi-Url", "https://localhost:" + Integer.toString(okapiPort));
-    httpClientMock = new HttpClientMock2(okapiHeaders.get("X-Okapi-Tenant"), okapiHeaders.get("X-Okapi-Url"));
-    try {
-      // Mocking the RM API Configuration response
-      httpClientMock.setMockJsonContent(MOCK_RMAPI_CONFIG_SUCCESS_FILE);
-    } catch (final IOException e) {
-      context.fail("Cannot read mock file: " + MOCK_RMAPI_CONFIG_SUCCESS_FILE + " - reason: " + e.getMessage());
-    }
-  }
-
-  @After
-  public void tearDown(TestContext context) {
-    logger.info("Codex Instances Resource Testing Complete");
-    vertx.close(context.asyncAssertSuccess());
   }
 
   @Test
   public void getCodexInstancesByIdSuccessTest(TestContext context) {
-    final Async asyncLocal = context.async();
```

Comment:
I removed calls to context.async because they don't seem to do anything, according to documentation "This method shall be used for creating asynchronous exit points for the executed test.". But in this case, asyncLocal.complete is called at the end of the test method, end of the method is a normal synchronous exit point, so we don't need to create another asynchronous exit point here